terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -742,18 +742,31 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 ]
             }
         },
+        "aws_s3_bucket": {
+            "gitlab": {
+                "bucket": f"edu-ucsc-gi-singlecell-hca-gitlab-{config.deployment_stage}",
+                "acl": "log-delivery-write"
+                # FIXME:  Bucket lacks lifecycle policy
+                #         https://github.com/DataBiosphere/azul/issues/3620
+            }
+        },
         "aws_lb": {
             "gitlab_nlb": {
                 "name": "azul-gitlab-nlb",
                 "load_balancer_type": "network",
                 "subnets": [
                     f"${{aws_subnet.gitlab_public_{zone}.id}}" for zone in range(num_zones)
                 ],
+                "access_logs": {
+                    "bucket": "${aws_s3_bucket.gitlab.id}",
+                    "prefix": "logs/nlb",
+                    "enabled": True
+                },
                 "tags": {
                     "Name": "azul-gitlab"
                 }
             },
-            "gitlab_alb": {
+            "gitlab_albƒ": {
```

Comment:
Did you test this at all?

Reply:
I had not, but I should have. I've [posted a summary of the tf-plan](https://github.com/DataBiosphere/azul/pull/3611#issuecomment-963503581) for the upgrades to the GL instance. If appears to be valid.